x/concentrated-liquidity/lp_test.go

```diff
@@ -551,6 +552,148 @@ func (s *KeeperTestSuite) TestisInitialPositionForPool() {
 	}
 }
 
+func (s *KeeperTestSuite) TestUpdatePosition() {
```

Comment:
nit: can you update the spec for `updatePosition` to remove the tests TODO please

Reply:
done.